scripts/fetch_tools.sh

```diff
@@ -164,6 +166,27 @@ get_controller-gen() {
     GOBIN=${DEST_DIR} GOFLAGS="" go install sigs.k8s.io/controller-tools/cmd/controller-gen@${ver}
 }
 
+get_robotframework() {
+    local rootdir
+    local venv
+
+    rootdir=$(git rev-parse --show-toplevel)
```

Comment:
The e2e runner does not move into the microshift root dir, so it does not have git. Can we use the ROOTDIR instead in line 176 and 186?